scripts/package

```diff
@@ -44,6 +44,7 @@ package (){
         docker run --rm --privileged multiarch/qemu-user-static:register --reset
         curl -sSL "https://github.com/multiarch/qemu-user-static/releases/download/${QEMUVERSION}/x86_64_qemu-${QEMUARCH}-static.tar.gz" | tar -xz -C tmp
         sed -i "s/CROSS_BUILD_//g" $dockerfilename
+        sed -i "s|QEMUARCH|$QEMUARCH|g" $dockerfilename
```

Comment:
1. where is `QEMUARCH` ? I can only see `ARCH` in https://github.com/rancher/ingress-nginx/blob/nginx-0.10.2-fix/images/nginx/Dockerfile
2. what is the difference between `sed |xx|yy` and `sed /xx/yy`?

Reply:
1. The package script is using https://github.com/rancher/ingress-nginx/blob/nginx-0.10.2-fix/rootfs/Dockerfile to build docker image.
2. Both \ and | can be used as separators in sed. Here is a Q&A in statckoverflow  https://stackoverflow.com/questions/33914360/what-delimiters-can-you-use-in-sed